plugins/hydra_submitit_launcher/hydra_plugins/hydra_submitit_launcher/submitit_launcher.py

```diff
@@ -96,36 +101,33 @@ def launch(
 
         num_jobs = len(job_overrides)
         assert num_jobs > 0
+        params = self.params
 
-        # make sure you don't change inplace
-        queue_parameters = self.queue_parameters.copy()
-        OmegaConf.set_struct(queue_parameters, True)
-        executors = {
-            QueueType.auto: submitit.AutoExecutor,
-            QueueType.slurm: submitit.SlurmExecutor,
-            QueueType.local: submitit.LocalExecutor,
-        }
-        init_parameters = {"cluster", "max_num_timeout", "slurm_max_num_timeout"}
-        executor = executors[self.queue](
-            folder=self.folder,
-            **{
-                x: y
-                for x, y in queue_parameters[self.queue.value].items()
-                if x in init_parameters
-            },
-        )
-        executor.update_parameters(
+        # build executor
+        init_params = dict(folder=params.submitit_folder)
```

Comment:
nit:
style wise, I have grown to prefer maps:
```suggestion
        init_params = {"folder" : params.submitit_folder}
```

Reply:
I used to always use maps, but recently changed my mind since now I find it's just more verbose (always requiring a string...). I'm fine with what you prefer anyway